src/qt/forms/optionsdialog.ui

```diff
@@ -199,10 +199,10 @@
        <item>
         <widget class="QCheckBox" name="allowIncoming">
          <property name="toolTip">
-          <string>Accept connections from outside</string>
+          <string>Accept connections from outside.</string>
          </property>
          <property name="text">
-          <string>Allow incoming connections</string>
+          <string>Allow incomin&amp;g connections</string>
```

Comment:
I don't see any of these being auto-assigned under Windows or Linux. Perhaps I'm missing something?